sdk/jdbc/azure-identity-extensions/pom.xml

```diff
@@ -10,12 +10,12 @@
   </parent>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-identity-providers</artifactId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-identity-providers;current} -->
+  <artifactId>azure-identity-extensions</artifactId>
```

Comment:
Should this package be in `sdk/identity` instead of `sdk/jdbc`? We expect this extensions package to have Redis, Mongo and other extensions too.

Reply:
Thanks for mentioning it, and thanks for pointing out where it should be.

Yes, I was thinking that as well. But I think it won't be a blocking issue for the GA release, so I plan to move it after the GA release.